src/dispatch/incident/flows.py

```diff
@@ -499,6 +510,10 @@ def incident_create_flow(*, incident_id: int, checkpoint: str = None, db_session
     )
 
     # create the incident ticket
+    incident_type_data = incident_type_service.get_by_name(
+        db_session=db_session, name=incident.incident_type.name
+    )
+
```

Comment:
Forgot to remove this one?

Reply:
yeah I updated it

Reply:
All good. Looks like I wasn't looking at the latest version. Thanks